src/Row.tsx

```diff
@@ -94,9 +79,9 @@ function Row<R, SR>(
           isCopied={copiedCellIdx === idx}
           isDraggedOver={draggedOverCellIdx === idx}
           isCellSelected={isCellSelected}
-          dragHandleProps={dragHandleProps}
-          onFocus={onFocus}
-          onKeyDown={onKeyDown}
+          dragHandle={isCellSelected ? selectedCellDragHandle : undefined}
+          onFocus={isCellSelected ? onFocus : undefined}
+          onKeyDown={isCellSelected ? onKeyDown : undefined}
```

Comment:
What if we pass `onFocus`/`onKeyDown` as is all the time? 🤔

Reply:
It would rerender all the cells then